virt/live_migration/virt-live-migration.adoc

```diff
@@ -13,5 +13,6 @@ include::modules/virt-about-live-migration.adoc[leveloffset=+1]
 == Additional resources
 
 * xref:../../virt/live_migration/virt-migrate-vmi.adoc#virt-migrate-vmi[Migrating a virtual machine instance to another node]
+* xref:../../virt/logging_events_monitoring/virt-monitor-vmi-migration.adoc#virt-monitor-vmi-migration[Monitoring live migration]
```

Comment:
Good to see that you added an xref to the moved assembly. That will be sufficient for the present! Thanks.